components/atoms/TocWrapper/index.tsx

```diff
@@ -20,26 +21,36 @@ interface Props {
 }
 
 const TocWrapper: FunctionComponent<Props> = props => {
-    const [elements, setElements] = useState<Element[] | null>(null)
+    const [headers, setHeaders] = useState<Element[] | null>(null)
```

Comment:
Changing the naming here, since in this file we are always talking about the headers.